terms.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Terms of Use"
+---
+
+## 1. Introduction
+
+These Terms of Use (the "Terms") govern your use of R-multiverse (the "Repository").
+By using the Repository, you agree to comply with these Terms.
+If you do not agree with these Terms, you may not use the Repository. 
+
```

Comment:
What about an "Acceptable use" section for users? From https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#c-acceptable-use:

> Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction. You are responsible for making sure that your use of the Service is in compliance with laws and any applicable regulations.

For example, what if a user is legally prohibited from downloading a package in their country even if we think the package itself is okay?

Reply:
I think we've covered by the general indemnity under the terms, and it is not for us to stipulate what a user does (and it is not reasonable for us to do so).

However, I've added a bullet to the indemnity covering the above, as it's not an exhaustive list anyway.

Reply:
> I think we're covered by the general indemnity under the terms, and it is not for us to stipulate what a user does (and it is not reasonable for us to do so).

Whether in this document or elsewhere, there are other aspects of acceptable use that we need to cover: for example, [violent](https://docs.github.com/en/site-policy/acceptable-use-policies/github-threats-of-violence-and-gratuitously-violent-content) or [obscene](https://docs.github.com/en/site-policy/acceptable-use-policies/github-sexually-obscene-content) content that is illegal to even own. A third-party contributor could upload illegal obscene images, and then they would be in our repository. Even if nobody downloads those images, we could still face prosecution.

We need to make it clear that the package maintainers have ownership and responsibility for content, not us, and that we are indemnified from the actions of contributors and package maintainers. The current document only indemnifies us from damages of people downloading packages.

> However, I've added a bullet to the indemnity covering the above, as it's not an exhaustive list anyway.

Thanks for that.

Reply:
This belongs in an Acceptable Use Policy, as you've noted elsewhere as well.